RpcNep5Tracker/RpcNep5Tracker.cs

```diff
@@ -192,7 +192,7 @@ public void OnPersist(Snapshot snapshot, IReadOnlyList<Blockchain.ApplicationExe
                     script = sb.ToArray();
                 }
 
-                using (ApplicationEngine engine = ApplicationEngine.Run(script, snapshot, extraGAS: maxGas))
+                using (ApplicationEngine engine = ApplicationEngine.Run(script, snapshot.Clone(), extraGAS: maxGas))
```

Comment:
This is for safety reasons, right?

Reply:
> This is for safety reasons, right?

Yes, beacuse some bugs were caused if not clone here.